### PR TITLE
Using gRPC with Go

### PR DESCRIPTION
**Pull request by tulikac**

### CODE REVIEW

File: HowTo/gRPC/Go/use_gRPC_with_go.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+You can run your code locally to ensure that it works. Once this is done your application is now ready to deploy to App Service.
+
+### Deploy to App Service
+Now that you have your server application setup and running locally, you can go to the portal and create your web app.  One thing to note is that Go and gRPC is currently only supported on Linux so be sure to choose this option when creating your web app.
+
+Create your web app as you normally would.  Choose **Code** as your Publish option.  Choose **Go 1.18 (Experimental)** or **Go 1.19 (Experimental)** as your Runtime stack and **Linux** as your Operating System.  
+
+Now that your web app is created, you'll need to do the following before deploying your application:
+
+#### 1. Enable HTTP version
```
Comment (btardif) [inline HowTo/gRPC/Go/use_gRPC_with_go.md:124]: @tulikac This should be *Set HTTP version*

**Comment by btardif** [comment]
@tulikac can you work through the comments and push an updated PR?

File: HowTo/gRPC/Go/use_gRPC_with_go.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
+# How-to deploy a Go gRPC app on App Service
+
+> [!WARNING]
+> gRPC is currently available to try as a Public Preview feature.
```
[the remark this answers is not in the document]
Comment (tulikac) [reply HowTo/gRPC/Go/use_gRPC_with_go.md]: This comment was about gRPC which is public preview. But I added a comment about Go as well.

File: HowTo/gRPC/Go/use_gRPC_with_go.md (hunk 1 above)
Comment (tulikac) [reply HowTo/gRPC/Go/use_gRPC_with_go.md]: Done

File: HowTo/gRPC/Go/use_gRPC_with_go.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+gRPC is a Remote Procedure Call framework that is used to streamline messages between your client and server over HTTP/2.  Using gRPC protocol over HTTP/2 enables the use of features like multiplexing to send multiple parallel requests over the same connection and bi-directional streaming for sending requests and responses simultaneously.  
+
+The following is a tutorial on how to deploy a Go gRPC application on App Service. 
+
+#### Prerequisite
+In this tutorial, we'll be deploying a gRPC server to App Service and making a gRPC request to the deployed server from a local gRPC client.  If you have not created a gRPC client and server yet, please follow this [Go Quick Start](https://grpc.io/docs/languages/go/quickstart/) to do so.  
+
+The following tutorial builds from the created gRPC client and server in that documentation.  If you already have a gRPC client and server, you may use these steps to add to existing Go apps as well.  This tutorial will work for both Go 1.18 and Go 1.19.
+
+### Setup the gRPC Server app
```
[the remark this answers is not in the document]
Comment (tulikac) [reply HowTo/gRPC/Go/use_gRPC_with_go.md]: Done

**Pull request merged**